namespaces/http.yml

```diff
@@ -0,0 +1,59 @@
+version: 2016.03.10.0
+
+defaults:
+  type: string
+  doc_values: true
+  index: not_analyzed
+
+namespace:
+  name: http
```

Comment:
Well...no?  I mean, these are things that are common to http requests, and are typically logged by http servers of any sort.  I didn't seem to make sense to place them under any "product specific" namespace.  Maybe apache, or nginx, or haproxy, or python standalone, or...

At least, that was my thinking.